USF-1670: Dropins Localization (#95)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/payment-services/payment-dictionary.mdx

---

> title: Payment Services Dictionary

Reviewer: ```suggestion
title: Payment Services dictionary
```

> description: Reference the default dictionary for the Payment Services dropin.

Reviewer: ```suggestion
description: Learn how to customize the default dictionary for the Payment Service drop-in component.
```

---

import CodeImport from '@components/CodeImport.astro';
import CodeInclude from '@components/CodeInclude.astro';

> The default Payment Services dictionary file (`i18n/en_US.json`) contains the default values for all labels and text common to the Payment Services drop-in.

Reviewer: ```suggestion
The default Payment Services dictionary file (`i18n/en_US.json`) contains the default values for all labels and text common to the Payment Services drop-in component.
```

Reviewer: I think this is an exception to our branding guidance because our Payment Services product is branded as a proper noun (capitalized).
